docs/configuration.md

```diff
@@ -122,6 +122,13 @@ fladle {
     outputStyle = 'multi'
     legacyJunitResult = false
     fullJunitResult = false
+    additional-apks:
+      - gs://path/to/app1.apk
```

Comment:
Hey, can we update the docs here to give an example of how to add `additional-apks` using the groovy syntax?
Don't forget to add the same thing in the kotlin syntax. :)

Can you also update the release notes to include the changes? `docs/changelog.md`? Thanks.